scripts/sanitycheck

```diff
@@ -1543,8 +1538,12 @@ class TestCase(object):
         warnings = None
 
         with open(inf_name) as inf:
-            with contextlib.closing(mmap.mmap(inf.fileno(), 0, mmap.MAP_PRIVATE,
-                                              mmap.PROT_READ, 0)) as main_c:
+            if os.name == 'nt':
+                mmap_args = {'fileno':inf.fileno(), 'length':0, 'access':mmap.ACCESS_READ}
+            else:
+                mmap_args = {'fileno':inf.fileno(), 'length':0, 'flags':mmap.MAP_PRIVATE, 'prot':mmap.PROT_READ, 'offset':0}
+
+            with contextlib.closing(mmap.mmap(**mmap_args)) as main_c:
```

Comment:
Maybe the `mmap()` stuff here could be removed. IIRC it caused problems with line terminators on Windows earlier before we fixed it, though it might've been in a different script.

Can get the contents of the file as a string with `inf.read()`. Suspect any performance difference might be negligible for this script.

Reply:
agree, and I am planning to remove that in a future patch.